app/hydrosheds_app.cc

```diff
@@ -1,35 +1,275 @@
-#include "hydrosheds/hydrosheds.hh"
+#include "../include/hydrosheds/hydrosheds.hh"
```

Comment:
This change should not be necessary. The `include` directory is added to the compiler invocation with a `-I` compile flag by CMake. Handling this through CMake has the advantage that it still works if e.g. `hydrosheds-cpp-api` is installed with `make install`, in which case relative paths might be invalidate.